docker-compose.latest.yml

```diff
@@ -85,7 +87,7 @@ services:
   chaosgenius-db:
     container_name: chaosgenius-db
     image: postgres:13
-    restart: always
+    restart: unless-stopped
     healthcheck:
       test: [ "CMD", "pg_isready", "-q", "-d", "chaosgenius_data", "-U", "postgres" ]
       timeout: 45s
```

Comment:
This one still has a `restart: always` on line 94

Reply:
@Samyak2 imo, we should update the docker-compose during the release time.